src/main/kotlin/br/com/fiap/carsocial/service/api/controller/response/RideResponse.kt

```diff
@@ -0,0 +1,21 @@
+package br.com.fiap.carsocial.service.api.controller.response
+
+import br.com.fiap.carsocial.service.api.document.Ride
+import br.com.fiap.carsocial.service.api.model.Car
+
+data class RideResponse(
+        var id: String? = null,
+        var name: String? = null,
+        var email: String? = null,
+        var car: Car?=null,
+        var coords: CoordsResponse?=null
+) {
```

Comment:
Você precisa utilizar constructor em kotlin, pois já está iniciando as variáveis

Reply:
É um data class, então esse outro construtor é opcional. Serve para construir um build response a partir de um Ride (document do mongodb). Recomenda outra forma de fazer isso?

Reply:
Pode passar o Objeto inteiro se é Igual, me parece que vc passa Ride para Response...
Menos código repetido!!!

`
data class RideResponse(
    var results: Ride = Ride()
)
override fun create(userRequest: RideRequest): Mono<RideResponse> {
        return rideRepository.save(userRequest.convertToRide())
            .map {
                RideResponse(it)
         }
}
`

Reply:
Entendi... mas isso não seria perigoso.. tipo se no futuro fosse adicionada alguma informação que não quero passar no response ao objeto Ride... se alguém esquecesse disso poderia ser um problema. Eu estaria passando um documento do mongo diretamente para o response. O que acha?